### PR TITLE
Restructured all of my script to operate within a 'for' loop.

### PR DESCRIPTION
**Pull request by npisciotti1**
Also created scenarios in which my users weren't cooperating.

Also restructured my HTML to use "section" tags instead of "div" elements where appropriate.

Added my top-10 goal list to my page.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,55 +1,102 @@
...
-if (question4 === 'YES') {
-  var question5 = prompt('Ok! Now that we\'ve been acquainted and know we have similar interests, do you think we can be friends ' + userName + '? YES or NO?').toUpperCase();
-  console.log('Ok! Now that we\'ve been acquainted and know we have similar interests, do you think we can be friends ' + userName + '?');
-  console.log(question5);
-} else if (question4 === 'NO') {
-  alert('Well either way, thank you for taking the time to visit my site. Take care ' + userName + '!');
-  console.log('Well either way, thank you for taking the time to visit my site. Take care ' + userName + '!');
+  } else if (i === 1) {
+    var q2 = prompt('Ok ' + userName + ', I\'m going to ask you some questions. First question, how old are you?');
+    console.log('Ok ' + userName + 'I\'m going to ask you some questions. First question, how old are you?');
```
Comment (nhuntwalker) [inline app.js]: There needs to be a Javascript `alert` after every prompt is answered

File: app.js [hunk 2]
```diff
@@ -1,55 +1,102 @@
...
+    console.log('How cool! Ok ' + userName + ', tell me your favorite hobby.');
+    console.log(q4);
+  } else if (i === 4) {
+    var q5 = prompt('Alright! Thanks for playing along thus far, ' + userName + '! In a moment you\'ll have a chance to guess something about me! But I have one more for you first. Name the city you live in!');
+    console.log('Alright! Thanks for playing along thus far, ' + userName + '! In a moment you\'ll have a chance to guess something about me! But I have one more for you first. Name the city you live in!');
+    console.log(q5);
+  } else if (i === 5) {
+    alert('Ok, now it\'s time for a guessing game! I want you to try and guess the number I\'m thinking of! Give it a go, but you only get four chances!');
+    console.log('Ok, now it\'s time for a guessing game! I want you to try and guess the number I\'m thinking of! Give it a go, but you only get four chances!');
+    for (var j = 1; j < 5; j++) {
```
Comment (nhuntwalker) [inline app.js]: Convention is to start loops at zero when you're just counting up.

File: app.js [hunk 3]
```diff
@@ -1,55 +1,102 @@
...
+    if (guessCorrect === false) {
+      alert('Uh oh! You didn\'t get the correct answer within 4 tries, it was ' + myFaveNum + '. Try the game over!');
+      console.log('Uh oh! You didn\'t get the correct answer within 4 tries, it was ' + myFaveNum + '. Try the game over!');
+      break;
+    }
+  } else if (i === 6) {
+    for (var k = 1; k < 7; k++) {
+      var q7 = prompt('Last question! I LOVE guitars! Can you guess what kind of guitar I have? I\'ll give you a hint, I have 3 guitars. The correct answer is a famous model of FENDER, GIBSON, or EPIPHONE. I only need the model name. Six attempts!');
+      console.log('Last question! I LOVE guitars! Can you guess what kind of guitar I have? I\'ll give you a hint, I have 3 guitars. The correct answer is a famous model of FENDER, GIBSON, or EPIPHONE. I only need the model name. Six attempts!');
+      if (isNaN(q7) && q7 !== null) {
```
Comment (nhuntwalker) [inline app.js]: `isNaN` returns true if the argument given is not a number. If the user gives valid input, the actual code block that you want to run won't run

File: app.js [hunk 4]
```diff
@@ -1,55 +1,102 @@
...
+        if (q7.toLowerCase() === guitarsOwned[0] || q7.toLowerCase() === guitarsOwned[1] || q7.toLowerCase() === guitarsOwned[2]) {
```
Comment (nhuntwalker) [inline app.js]: What's something you could do to not list out every indexed part of your array?

File: app.js (hunk 2 above)
Comment (npisciotti1) [inline app.js]: Cool, that's been changed!

File: app.js (hunk 4 above)
Comment (npisciotti1) [inline app.js]: for-loop!  As for all of these changes above, would you like me to resubmit this commit of the code separately? My final "about me" incorporates many similarities to the suggestions you've made. In other words, I can resubmit the older version for the sake of the assignment, but my newer code has some of these changes and I can also update it (for example my for-loop index starting at 0).

**Pull request merged**